day04/index.js

```diff
@@ -0,0 +1,10 @@
+const { passwordFinder1 } = require('./passwordFinder1');
+const { passwordFinder2 } = require('./passwordFinder2');
+
+// Retrieve acceptable password options by calling either methods with a start and end value.
+
+// getPasswordsFilter filters through a generated array of all possibilites within range.
+const getPasswordsFilter = passwordFinder1;
+
+// getPasswordsLoop leverages a for loop to generarte the array of acceptable passwords.
```

Comment:
mention performance stuff if you want!

Reply:
although one side note: I generally prefer the comments for a function like with that function in the file itself

Reply:
*I can't seem to edit my comment, but I meant: "although one side note: I generally prefer the comments explaining a function to go in the same file alongside the actual function logic"